proto/dictation_asr.proto

```diff
@@ -418,46 +506,110 @@ message StreamingRecognizeResponse {
     // until the server closes the gRPC connection. This event is only sent if
     // `single_utterance` was set to `true`, and is not used otherwise.
     END_OF_SINGLE_UTTERANCE = 1;
-    
+
+    // [**Extension by Techmo**]
+    // This event indicates that recognition has started and there was no speech detected
+    // for a certain period of time. The server will not process
+    // additional audio (although it may subsequently return additional results).
+    // The client should stop sending additional audio data, half-close the gRPC
+    // connection, and wait for any additional results until the server closes
+    // the gRPC connection.
+    // MRCPv2-related: [No-Input-Timeout](https://tools.ietf.org/html/rfc6787#section-9.4.6)
+    NO_INPUT_TIMEOUT = 2;
```

Comment:
Probably we should remove 'Timeout' from all fields in this `SpeechEventType` enum. `SpeechEvent::NO_INPUT` sounds better than `NO_INPUT_TIMEOUT`.